### ISSUE TITLE
Continuously send artnet values

### ISSUE THREAD

**Issue by jk-5** [jnimmo/hass-artnet#5]
Hi, Some Artnet interfaces require values to be sent continiously to be able to keep the lights on. For HTP operation, some interfaces have an timeout per sending host, and will forget all values if this timeout is reached. This will cause all lights to go to black a few seconds after sending the last value. Sending all values continiously every second will fix this. The full 40 fps is not required if nothing changes.
I'd love to have this feature enabled by default or able to enable via a config option.

**Comment by matteofranceschini** [comment]
Yeah, this could be useful.. Although, I've solved the issue by adding a "virtual" light on an unused channel, linked with an automation that toggles it every five seconds or so!

**Comment by martin3000** [comment]
I have a professionall stage light, a moving head wash light from lixada and changing a value, the lights go on for a second then off again. It definitly needs continuous values.
